README.md

```diff
@@ -42,6 +42,8 @@ Once that command is running you have to create the iamservice account so that p
 Make sure to change the namespace if you are not deploying in `kube-system`
 
 ```
+eksctl utils associate-iam-oidc-provider --region=us-east-1 --cluster=paul-eks --approve // The IAM OIDC Provider is not enabled by default on EKS cluster, you can use this command to enable it.
```

Comment:
lets remove eks-paul and make that a default clustername they should replace with their eks cluster name. same for the 2nd command